src/locals/hu.json

```diff
@@ -0,0 +1,145 @@
+    "emotional_reasoning_explanation_2": "Ezt sokszor nehéz érzékelni. Nehéz összeszedned magad annyira, hogy logikusan tudj gondolkozni az érzelmeidről is.",
+    "emotional_reasoning_thought": "Bűnösnek érzem magam, ezért biztos, hogy rosz fát tettem a tűzre.",
+    "fortune_telling": "Jövendőmondás",
+    "fortune_telling_explanation": "Amikor ki akarjuk találni a jövőt. Sokszor túlbecsüljük a jövendőmondó képességünket. Sokszor erős előfeltételezésekkel élünk és csak az azt alátámasztó bizonyítékokat vesszük figyelembe.",
+    "fortune_telling_thought": "Le fog zuhanni a repülő, amin ülök.",
+    "labeling": "Címkézés",
+    "labeling_explanation": "Ha egy jellegzetes tulajdonsággal azonosítunk egy embert, akkor címkézünk. Ha valaki bevágott eléng, attól még nem biztos, hogy ő \"*eggfej,\" lehet, hogy csak siet. Ez ránk is vonatkozik; csak azért, mert hibázunk még nem vagyunk \"hibásak.\"",    
+    "labeling_thought": "Elrontottam a ZH-t/TZ-t, biztos rossz tanuló vagyok.",
+    "magnification_of_the_negative": "A Negatívumok felerősítése",
+    "magnification_of_the_negative_explanation": "Ha figyelmen kívül hagyod a sikereket, akkor lehet, hogy \"rontóc\" vagy. Hogyha folyton az elbaltázott interjúkon rágódsz, akkor lehet, hogy nem jut eszedbe, hogy mennyi új tudásra tettél szert általuk.",
+    "magnification_of_the_negative_thought": "Figyeltem a kajálásra a héten, de kihagytam a futást.",
+    "mind_reading": "Gondolatolvasás",
+    "mind_reading_explanation": "Ha azon rágódunk, hogy mások mit gondolnak rólunk, akkor valószínűleg gondolatot szeretnénk olvasni. Ameddig nem mondják meg, hogy mit gondolnak rólunk, addig nem tudhatjuk, hogy tényleg az-e a helyzet, amit mi feltételeztünk. Miért is kéne mindig a legrosszabbra felkészülni?",
+    "mind_reading_thought": "Bunkó voltam Gyurival, mert nem mentem el a bulijára, most biztos utál engem.",
+    "minimization_of_the_positive": "A Sikerek elhallgatása",
+    "minimization_of_the_positive_explanation": "Ha lebecsüljük a sikereinket, akkor minimalizáljuk a (dopamint-tüzelést) [ez hogy mondjuk magyarul? ¿Öröm?]. Akkor is, ha az első napunk nem ment 100%-ig a tervnek megfelelően, nem jelenti azt, hogy pl. 60% ami jól ment azt ignorálhatjuk.",
+    "minimization_of_the_positive_thought": "Sok embernek tetszett az előadásom, de elbaltáztam a bevezetést, szóval biztos elcsettintettem.",
+    "other_blaming": "Mások hibáztatása",
+    "other_blaming_explanation_1": "Ha egy rossz helyzetet mindig valaki másnak a hibájának gondolunk, akkor másokat hibáztatunk. Ha megbuktál egy vizsgán és a tanárt hibáztatod, akkor rossz irányba feccölöd a megoldásra szánt energiáidat. Valaki eléd vágott, az autópályán? Ha rádudálsz, rátapadsz, az segít? Most közveszélyes és ideges is vagy egyszerre!",
```

Comment:
"közveszélyes" -> I think in the original, it was the other person who was the jerk, the point being "the only thing you can change is whether you're mad or not". Maybe.. "Most már úgyis előtted van, de ráadásul még ideges is vagy miatta."